images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -1,3 +1,5 @@
+Import-Module "$PSScriptRoot/../helpers/SoftwareReport.Helpers.psm1" -DisableNameChecking
```

Comment:
Why do we need to import in any file?
It is already imported in `SoftwareReport.Generator.ps1`. It should be enough

Reply:
These files are using functions from `/helpers/SoftwareReport.Helpers.psm1`.